bin/ledger2beancount-txns

```diff
@@ -413,9 +413,10 @@ while (my $l = <>) {
 	$l =~ s/$account/$beancount_account/;
 
 	if ($l =~ /$amount_RE/) {
+	    my $num = $+{num};
 	    my $commodity = $+{commodity};
 	    my $beancount_commodity = map_commodity $commodity;
-	    $l =~ s/$commodity/$beancount_commodity/;
+	    $l =~ s/\Q$num\E\s*$commodity/$num $beancount_commodity/;
```

Comment:
i think the `\s*` should be a `\s+` here, otherwise you might match without a space --- that might already be wrong per se, but it's at least inconsistent with the fact you non conditionally add a space on the right hand side of this replacement

Reply:
the correct quitespace is covered by `$amount_RE` anyway so it doesn't really matter.  But your comment made me realized I could just re-use `$amount_RE`.  In fact, it could be much simpler.